### ISSUE TITLE
geom_abline doesn't work

### ISSUE THREAD

**Issue by cpsievert** [tdhock/animint#92]
As [noted here](https://github.com/tdhock/animint/pull/84#issuecomment-112899816), `geom_abline()` isn't working. Here is a mini-example:

```r
library(animint)
p <- qplot(wt, mpg, data = mtcars) + geom_abline(intercept = 20)
animint2dir(list(p = p))
```

**Comment by kferris10** [comment]
Fixed in #93

**Comment by kferris10** [comment]
Re-opening.  Only works for one abline
